plugins/unifying/lu-device.h

```diff
@@ -64,7 +64,8 @@ struct _LuDeviceClass
 
 #define LU_DEVICE_EP1				0x81
 #define LU_DEVICE_EP3				0x83
-#define LU_DEVICE_TIMEOUT_MS			2500
+/* Signed firmware are very long to verify on the device */
+#define LU_DEVICE_TIMEOUT_MS			20000
```

Comment:
This is fine, but do we set the device status to `FWUPD_STATUS_DEVICE_BUSY` when this is happening?

Reply:
Wasn't this already done? Even when the timeout was only 2.5 seconds?